admin/admin-domain-service/ballerina/resources/admin-api.yaml

```diff
@@ -1146,7 +1146,7 @@ paths:
   ######################################################
   # The "Individual Deny Policy" resource API
   ######################################################
-  /deny-policy/{conditionId}:
+  /deny-policies/{conditionId}:
```

Comment:
shouldnt we use policyID for this one. As we are discussing about policies ID should be policy id.

Reply:
resolved by changing to policyId